### PR TITLE
Publisher: Add new publisher to host tools

### PR DESCRIPTION
**Pull request by iLLiCiTiT**
## Brief description
Added new publisher to host tools. Code copied from https://github.com/pypeclub/OpenPype/pull/3046 which is in draft and the code is already needed for other PRs.

## Description
Added new publisher to host tools to be able easily open the tool from hosts.

## Testing notes:
1. Calling `show_publisher` in host tools should work

```
from openpype.tools.utils.host_tools import show_publisher

show_publisher()
```

### CODE REVIEW

File: openpype/tools/utils/host_tools.py [hunk 1]
```diff
@@ -269,6 +271,30 @@ def show_experimental_tools_dialog(self, parent=None):
             dialog.activateWindow()
             dialog.showNormal()
 
+    def get_publisher_tool(self, parent):
+        """Create, cache and return scene inventory tool window."""
+        if self._scene_inventory_tool is None:
```
Comment (BigRoy) [inline openpype/tools/utils/host_tools.py:276]:
```suggestion
        if self._publisher_tool is None:
```

Comment (iLLiCiTiT) [reply openpype/tools/utils/host_tools.py:276]: You're faster then Sonic

**Review by antirotor — APPROVED** [review]
It worked in the original PR so I think it should work here too :)

File: openpype/tools/utils/host_tools.py [hunk 2]
```diff
@@ -205,6 +206,7 @@ def show_publish(self, parent=None):
         pyblish_show = self._discover_pyblish_gui()
         return pyblish_show(parent)
 
+
```
Comment (BigRoy) [inline openpype/tools/utils/host_tools.py:209]: Shouldn't the hound complain about these?
```suggestion

```

File: openpype/tools/utils/host_tools.py [hunk 3]
```diff
@@ -269,6 +271,31 @@ def show_experimental_tools_dialog(self, parent=None):
             dialog.activateWindow()
             dialog.showNormal()
 
+    def get_publisher_tool(self, parent):
+        """Create, cache and return scene inventory tool window."""
```
Comment (BigRoy) [inline openpype/tools/utils/host_tools.py:275]:
```suggestion
        """Create, cache and return publisher window."""
```

**Pull request merged**